### PR TITLE
global: installation slimdown

### PR DESCRIPTION
**Pull request by crepererum**
Slimdown installation. See individual commits for explanation.

Rebased against: #3371

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -160,8 +159,6 @@ def run(self):
     "dropbox": [
         "dropbox>=2.1.0"
     ],
-    "elasticsearch": [
```
Comment (jirikuncar) [inline setup.py]: you should keep it for empty for few versions not to break overlays depending on Invenio.

File: setup.py [hunk 2]
```diff
@@ -175,9 +172,6 @@ def run(self):
         "qrcode>=5.1",
         "Pillow>=2.7.0"
     ],
-    "mongo": [
-        "pymongo>=3.0"
```
Comment (jirikuncar) [inline setup.py]: just make it empty

**Comment by crepererum** [comment]
done

**Comment by jirikuncar** [comment]
`s/remove/removal/`

**Comment by crepererum** [comment]
done

**Comment by jirikuncar** [comment]
:shipit:

**Pull request merged**